Source/engine/render/scrollrt.cpp

```diff
@@ -1322,23 +1320,24 @@ void DrawView(const Surface &out, Point startPosition)
  */
 void DrawFPS(const Surface &out)
 {
-	char buf[12];
+	static int framesSinceLastUpdate = 0;
+	static fmt::basic_memory_buffer<char, 12> buf {};

```

Comment:
I'm not sure if there's a way to specify a void allocator to prevent this from ever resizing? We can make this work with a static string_view and static char[] buffer instead of fmt::memory_buffer if that's desirable.

Reply:
I think a static char buffer would be slightly simpler since it avoids all the capacity checks of `basic_memory_buffer`.

Reply:
done :)